src/main/java/roomescape/reservationTime/dto/AllReservationTimeResponse.java

```diff
@@ -0,0 +1,14 @@
+package roomescape.reservationTime.dto;
+
+import java.util.List;
+import roomescape.reservationTime.ReservationTime;
+
+public record AllReservationTimeResponse(List<ReservationTimeResponse> reservationTimes) {
```

Comment:
```suggestion
public record ReservationTimeResponses(List<ReservationTimeResponse> reservationTimes) {
```
사소한 부분이지만 클래스명에 All을 붙이지 않아도 ReservationTimeResponse 목록을 담는 DTO라는 의미가 충분히 전달될 것 같아요.
또 모든 목록이 아닌 특정 조건에 해당하는 ReservationTimeResponse 목록을 반환해야 하는 요구사항이 생기더라도 이 클래스를 재사용할 수 있을 것 같아요.

Reply:
오호!! 그러네요! 감사합니다 👍🏻